index.html

```diff
@@ -1925,37 +1925,53 @@ <h4>Error Responses</h4>
           </span>
         </p>
       </section>
-    </section>
+      </section>
+
+
+
 
     <!-- External Representation -->
     <section id="external-representation">
       <h2 id="external-td-representations">External TD
         representations</h2>
       <p>
-        <span class="rfc2119-assertion" id="profile-5.3-exteanal-td-representations-1">
+        <span class="rfc2119-assertion" id="profile-5.3-external-td-representations-1">
         The default representation is JSON. Semantic
         annotations based on JSON-LD MAY be present but are not
         required to perform all interactions with the thing
         instance.
         </span>
       </p>
 
-        <section id="canonical-representation">
-          <h2 id="canonical-td-representations">Canonical TD
-            representation</h2>
-          <p>
-            A canonical representation serves multiple purposes.
-            It is simplifying the parsing process, enables to identify
-            equivalent TDs by simple string comparisons.
-            Furthermore it allows the use of a simple signing mechanism,
-            such as <a href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs</a> or JSON Web Signatures [[RFC7515]] and enables
-            identity checks on encrypted TDs.
-          </p>
-	  <p>
-	     The canonical JSON representation format of a TD adopts the JSON Canonicalization Scheme (JCS) defined by
-	     <a href="https://tools.ietf.org/html/rfc8785>RFC-8785</a> including errata <a href="https://www.rfc-editor.org/errata/eid6292">6292</a>.
-	  </p>
-      </section>
+      <h3 id="character encoding">Character Encoding</h3>
```

Comment:
Nit: I think this heading needs to go inside a `<section>` element in order to get its own section number.